boards/arm/b_u585i_iot02a/board.cmake

```diff
@@ -12,7 +12,6 @@ if(CONFIG_BUILD_WITH_TFM)
   endif()
 endif()
 
-board_runner_args(stm32cubeprogrammer "--erase" "--port=swd" "--reset-mode=hw")
```

Comment:
Please check why it was there in the first place an explain why it is safe removing.